docs/whats_new/v0.1.1.md

```diff
@@ -0,0 +1,57 @@
+# Version 0.1.1
+
+We're happy to announce the AutoGluon-Cloud 0.1.1 release. This release added support for TimeSeriesCloudPredictor.
+Don't wait to try it out and we are happy to any feedback!
+
+Currently, AutoGluon-Cloud supports training and deploying TabularPredictor, MultiModalPredictor and TimeSeriesPredictor with AutoGluon Deep Learning Containers version 0.6.0 and newer.
+It is always recommended to use the latest version of the container as it has more features and are updated with security patches.
+
+To learn more, check out our [tutorial](https://auto.gluon.ai/cloud/0.1.1/index.html)
+
+This release contains **27** commits from **2** contributors.
+
+See the full commit change-log here: https://github.com/autogluon/autogluon-cloud/compare/0.1.0...0.1.1
+
+Full Contributor List (ordered by # of commits):
+- @yinweisu, @gidler
+
+This release supports Python versions 3.8, 3.9, and 3.10. Python 3.7 is no longer supported as of this release.
```

Comment:
You might want to update this: https://github.com/autogluon/autogluon-cloud/blob/ebbb2ec5e6b9e202bd7a527d02fe9b0edd325dde/pyproject.toml#L3 as well?

Reply:
Oh, this is a gooood reminder. Thanks!

Reply:
Very good catch!